### PR TITLE
Add a "what's express popover"

### PR DESCRIPTION
**Pull request by gadenbuie**
Adds a "what's express" popover to Express/Core tabsets

@gregswinehart this is ready for style nudges!

https://github.com/posit-dev/py-shiny-site/assets/5420529/c60d1a49-b593-47d7-b700-22c2ccb27f87

### CODE REVIEW

**Comment by gregswinehart** [comment]
Thank you sir! I will mess with it shortly.

File: include-in-header.html [hunk 1]
```diff
@@ -72,6 +72,35 @@
     })
   }
 
+  function addWhatsShinyExpressTooltip() {
+    const tooltipContents = `<p><strong>Shiny Express</strong> is a new, streamlined way to write a Shiny app.</p>
+    <p><strong>Shiny Core</strong> refers to the original, functional Shiny syntax, which is still a great way to write Shiny apps.</p>
+    <p class="text-right" style="font-size:0.85em;"><a href="docs/express-motivation.qmd#">Read more<i class="bi bi-chevron-right"></i></a></p>`
```
Comment (gadenbuie) [inline include-in-header.html:78]:
```suggestion
    <p class="text-right" style="font-size:0.85em;"><a href="/docs/express-introduction.qmd">Read more<i class="bi bi-chevron-right"></i></a></p>`
```

File: include-in-header.html [hunk 2]
```diff
@@ -72,6 +72,35 @@
     })
   }
 
+  function addWhatsShinyExpressTooltip() {
+    const tooltipContents = `<p><strong>Shiny Express</strong> is a new, streamlined way to write a Shiny app.</p>
+    <p><strong>Shiny Core</strong> refers to the original, functional Shiny syntax, which is still a great way to write Shiny apps.</p>
+    <p class="text-right" style="font-size:0.85em;"><a href="/docs/express-introduction.qmd">Read more<i class="bi bi-chevron-right"></i></a></p>`
```
Comment (gadenbuie) [inline include-in-header.html:78]: This link will need to be updated

**Comment by gadenbuie** [comment]
@gregswinehart I'm going to merge this into `rc-express` so that it's part of our previews. Your style nudges are still welcome!

**Pull request merged**